charts/genai-toolkit/templates/ui.yaml

```diff
@@ -26,6 +26,14 @@ spec:
               value: "AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://azurite-lb.default.svc.cluster.local:10000/devstoreaccount1;QueueEndpoint=http://azurite-lb.default.svc.cluster.local:10001/devstoreaccount1;TableEndpoint=http://azurite-lb.default.svc.cluster.local:10002/devstoreaccount1;"
             - name: AZURE_STORAGE_QUEUE_NAME
               value: "debug-log"
+            - name: AWS_STORAGE_CONNECTION_STRING
+              value: "http://host.docker.internal:4566/000000000000/debug-log"
+            - name: AWS_REGION
+              value: "us-east-1"
+            - name: AWS_ACCESS_KEY_ID
+              value: "test"
+            - name: AWS_SECRET_ACCESS_KEY
```

Comment:
Not sure if you want to include this now since your PR is already quite big, but my [helm PR for genai-toolkit ](https://github.com/NetAppLabs/genai-toolkit/pull/1642/files#diff-f60858aa44a778ac9bf51637a547bd5fea9a602b1ffa621433e5ac3f305813c1R57) had some improvements here with some other fixes

Reply:
Going to resolve this now because I know that it's a bad idea to try to get that work into here - this is more of a note to myself to fix my PR 😀